src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -391,9 +391,13 @@ String generateDeclarations(
       List<SourceFile> sourceFiles, List<SourceFile> externs, Depgraph depgraph)
       throws AssertionError {
     compiler.compile(externs, sourceFiles, opts.getCompilerOptions());
-    precomputeChildLists();
-    collectTypedefs();
-    String dts = produceDts(depgraph);
+    String dts = "";
```

Comment:
initialize to null? That's more idiomatic, I guess? Do we want to return an empty string or null from this for errors?